src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -129,4 +132,7 @@
     <Compile Include="JsonObjectTests.cs" />
     <Compile Include="JsonStringTests.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="Serialization\CustomConverterTests.DictionaryInt32StringKeyValueConverter.cs" />
```

Comment:
Why is this file added as a separate group with "None"?

Reply:
Good catch. Not sure but will fix.